### PR TITLE
Reuse unit tests for amount and year pages

### PR DESCRIPTION
**Pull request by pcraig3**
Allows us to remove a bunch of lines from the bottom of this file.

### CODE REVIEW

**Comment by pcraig3** [comment]
Robert Galanakis: 

> “The fastest code is the code which does not run. The code easiest to maintain is the code that was never written.”

File: routes/login/login.spec.js [hunk 1]
```diff
@@ -653,24 +665,8 @@ questionsAmounts.map(amountResponse => {
...
-            taxReturnYear: '2018',
-            taxReturnAmount: goodAmount,
-            redirect: '/start',
-          })
-        expect(response.statusCode).toBe(302)
-        expect(response.headers.location).toEqual('/start')
-      })
-    })
-
-    const goodYears = ['2018', '2017']
```
Comment (brdunfield) [inline routes/login/login.spec.js]: Was there a reason we originally tested against 2 years?

Comment (pcraig3) [reply routes/login/login.spec.js]: It was rewritten from an earlier check where only 2 values were accepted.

We accept tons of year values so these two are arbitrary.

Comment (pcraig3) [reply routes/login/login.spec.js]: Okay, you win.

**Comment by pcraig3** [comment]
image

**Pull request merged**